Problem1.py

```diff
@@ -0,0 +1,42 @@
+// Time Complexity : O(1) 
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Time limit exceeded
+// Any problem you faced while coding this : No
```

Comment:
Optimize on space and choose `self.size` value strategically.  Please try to Implement double hashing  or other techniques to improve the time usage.